support-workers/build.sbt

```diff
@@ -69,10 +89,26 @@ deployToCode := {
     "-FailureHandlerLambda-",
     "-SendAcquisitionEventLambda-",
     "-PreparePaymentMethodForReuseLambda-",
-  ).foreach { functionPartial =>
+  ).par.foreach { functionPartial =>
     log.info("updating " + functionPartial + "...")
-    s"aws lambda update-function-code --function-name support${functionPartial}CODE --s3-bucket $s3Bucket --s3-key $s3Path --profile membership --region eu-west-1".!!
+    log.info(s"aws lambda update-function-code --function-name support${functionPartial}CODE --s3-bucket $s3Bucket --s3-key $s3Path --profile membership --region eu-west-1".!!)
+    log.info(s"aws lambda update-alias --function-name support${functionPartial}CODE --name SNAPSTART --function-version ${'$'}LATEST --profile membership --region eu-west-1".!!)
     log.info("finished " + functionPartial)
   }
+}
 
+lazy val deployToCodeWithCfn =
+  inputKey[Unit]("Directly update CFN and AWS lambda code from CODE instead of via RiffRaff for faster feedback loop")
+
+deployToCodeWithCfn := {
+  import scala.sys.process._
+  val log = streams.value.log
+  val version = lambdaVersion.value
+  val s3Bucket = "support-workers-dist"
+  val s3Path = s"support/CODE/support-workers/support-workers-$version.jar"
+  val assemblyJar = assembly.value
+  log.info(s"generated jar $assemblyJar, about to upload to S3...")
+  log.info((s"aws s3 cp $assemblyJar s3://" + s3Bucket + "/" + s3Path + " --profile membership --region eu-west-1").!!)
+  log.info("now running scripts/update-code-cloudformation.sh in order to activate the uploaded JAR")
+  log.info((s"/bin/bash ./support-workers/scripts/update-code-cloudformation.sh $version").!!)
```

Comment:
we pass in the version to this script, which can then pass it in turn to the node script